schema.sql

```diff
@@ -0,0 +1,11 @@
+/* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+    id INTEGER,
+    name VARCHAR(50),
+    date_of_birth DATE,
+    secape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
+);
```

Comment:
[OPTIONAL] - I will advise that you mark the `id` column as primary key

```sql
PRIMARY KEY (id)
```

A primary key is used to ensure data in the specific column is unique, it uniquely identifies each record in a table

👉 [SQL PRIMARY KEY Constraint](https://www.w3schools.com/sql/sql_primarykey.ASP)